src/styles/blocks/container.scss

```
@@ -0,0 +1,40 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-auto-rows: 300px;
+
+  @media(min-width: 900px) {
+    grid-template-columns: auto auto auto;
+  }
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: white;
+
+  line-height: 300px;
+  text-align: center;
+
+  &__block {
+    width: 100%;
+    height: 100%;
+
+    &:first-child {
+      background-color: rgb(255, 0, 0);
+    }
+    &:nth-child(2) {
+      background-color: rgb(204, 0, 0);
+    }
+    &:nth-child(3) {
+      background-color: rgb(153, 0, 0);
+    }
+    &:nth-child(4) {
+      background-color: rgb(102, 0, 0);
+    }
+    &:nth-child(5) {
+      background-color: rgb(51, 0, 0);
+    }
+    &:nth-child(6) {
+      background-color: rgb(0, 0, 0);
+    }
```

```suggestion
    $colors:
    "1" rgb(255, 0, 0),
    "2" rgb(204, 0, 0),
    "3" rgb(153, 0, 0),
    "4" rgb(102, 0, 0),
    "5" rgb(51, 0, 0),
    "6" rgb(0, 0, 0);

  @each $number, $color in $colors {
    &--#{$number} {
      background-color: $color;
    }
  }
```
Что если мы сделаем вот так